src/main/scala/org/constellation/API.scala

```diff
@@ -226,6 +227,19 @@ class API(udpAddress: InetSocketAddress)(implicit system: ActorSystem, val timeo
 
   private val postEndpoints =
     post {
+      pathPrefix("channel") {
+        path("open") {
+          entity(as[ChannelOpenRequest]) { request =>
+            ChannelMessage.createGenesis(request)
+            complete(StatusCodes.OK)
```

Comment:
StatusCode 201 (Created) might be better, though doesn't really matter.

Reply:
Will fix